earthquakes.py

```diff
@@ -28,34 +28,57 @@ def get_data():
     # to a file and open it in VS Code or a browser.
     # See the README file for more information.
     ...
+    import json
+    with open('earthquakes_data.json', 'w') as f:
+        f.write(text)   # On Mac Shift + Option + F to format automatically
+    
+    with open('earthquakes_data.json', 'r') as f:
```

Comment:
@yihuitong00 good advise, but I think we still need these lines to load data as a dict from json file